app/views/users/confirmations/confirmed.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="govuk-grid-row">
+  <h2 class="govuk-heading-m confirmed-banner">Your school is registered</h2>
```

Comment:
Is this header gonna be full-width? Do we want to wrap it in a div saying so? 
```
<div class="govuk-grid-column-full-width">
```

Or should it be inside the two-thirds column?

Reply:
It will be full width